include/deal.II/base/parameter_handler.h

```diff
@@ -1063,7 +1063,8 @@ class ParameterHandler : public Subscriptor
   declare_entry(const std::string &          entry,
                 const std::string &          default_value,
                 const Patterns::PatternBase &pattern = Patterns::Anything(),
-                const std::string &          documentation = "");
+                const std::string &          documentation = "",
+                const bool                   has_to_be_set = false);
```

Comment:
You'll have to add documentation for this parameter. (There is also text above that talks about "The last parameter..." which is now no longer correct.)